### PR TITLE
OCD-3432: Add curesUpdate field to XML download; valid XSD/XML

### PR DESCRIPTION
**Pull request by kekey1**

### CODE REVIEW

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/Developer.java [hunk 1]
```diff
@@ -58,7 +58,7 @@
     /**
      * Indication of whether a health IT developer is a "self-developer" or not.
      */
-    @XmlElement(required = true)
+    @XmlElement(required = true, nillable = true)
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/Developer.java]: The data-model defines this as `self_developer bool NOT NULL DEFAULT false,`. Should it be nillable?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/TestProcedure.java [hunk 2]
```diff
@@ -2,14 +2,36 @@
...
 
-    public TestProcedure() {
-        super();
-    }
+    private Long id;
+    private String name;
 
-    public TestProcedure(Long id, String name) {
-        super(id, name);
+    public Long getId() {
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/TestProcedure.java:25]: Could these be handled with the `@Data` Lombok annotation? I have a vague memory that our XML things can't use those annotations for some reason, but wanted to double check

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DeveloperManager.java [hunk 3]
```diff
@@ -70,6 +70,7 @@
 @Log4j2
 public class DeveloperManager extends SecuredManager {
     public static final String NEW_DEVELOPER_CODE = "XXXX";
+    private static final Integer FIVE_SECONDS = 5000;
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DeveloperManager.java]: Maybe `DELAY_BEFORE_JOB_START` so there's meaning behind the magic number?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/TestProcedure.java (hunk 2 above)
Comment (kekey1) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/TestProcedure.java:25]: I also recall that we can't use @Data in classes that are involved in the XSD/XML stuff. That's why I didn't.

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/Developer.java (hunk 1 above)
Comment (kekey1) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/Developer.java]: Fixed. This was leftover from an attempt to avoid refactoring ProductDTO.

**Review by andlar — DISMISSED** [review]

**Review by tmy1313 — DISMISSED** [review]

**Review by andlar — APPROVED** [review]

**Review by tmy1313 — APPROVED** [review]

**Pull request merged**